src/Accelerators/NNPA/zdnn.cmake

```diff
@@ -1,7 +1,12 @@
 # SPDX-License-Identifier: Apache-2.0
 
 function(setup_zdnn version)
-  set(ZDNN_GITHUB_URL https://github.com/IBM/zDNN)
+  # Set policy CMP0097 to NEW for it to not initialize submodules
+  cmake_policy(SET CMP0097 NEW)
```

Comment:
This was not in the onnx-mlir/main branch, inherited from other repo. I cannot tell if it is needed or not.